packages/core/.eslintrc.json

```diff
@@ -2,12 +2,17 @@
   "root": true,
   "parser": "@typescript-eslint/parser",
   "plugins": ["@typescript-eslint", "import"],
+  // required for no-floating -promises lint rule
+  "parserOptions": {
+    "project": "tsconfig.linter.json" 
+  },
   "rules": {
     "@typescript-eslint/no-explicit-any": "off",
     "@typescript-eslint/ban-ts-ignore": "off",
     "@typescript-eslint/ban-ts-comment": "off",
     "import/no-unresolved": "error",
-    "import/default": "off"
+    "import/default": "off",
+    "@typescript-eslint/no-floating-promises": "error"
```

Comment:
What about the other non-core packages?  Do we need to add this rule to the `.eslintrc.json` for each of the packages?

Reply:
My plan os to create a set of three PRs ( easy reviewing purposes), they will cover all the packages. 1 is for core which is this one, 1 will be for packages that do err when linter is on with 1 change, 1 will be for 4 packages with smaller errors on adding the linter.

Reply:
To answer your first question we need to this rule for all the eslint.json files in all the packages